README.md

```diff
@@ -7,90 +7,278 @@
-- Explain pulling branches
+- Explain getting updates from remote
 - Explain merging branches
 - Explain deleting branches
 
 ## Introduction
 
-We're faced with many choices in life; some of these choices are easier, more convenient, or "safer" than others: Move away from familiarity or experiment with moving away and residing in a brand new city? Choose the usual ramen dish, or try it extra spicy? Style your hair the same or try cutting it all off into a radically new look?
+We're faced with many choices in life; some of these choices are easier, more
+convenient, or "safer" than others.
 
-Consider a computer upgrade. We have the option of buying a new computer, a used computer, or maybe there's nothing "wrong" with our current computer&emdash;we want more storage space. 
+Consider a computer upgrade. We have the option of buying a new computer, a used
+computer, or maybe there's nothing "wrong" with our current computer&emdash;we
+want more storage space.
 
-The cheapest option is most likely to buy compatible new parts for our existing computer. However, we've never attempted tinkering with hardware in our computer before, and after buying all the tools and parts, we follow an instructional video to make the upgrades. In this case, we only have one computer to work on, and nowhere to test out opening up a computer and connecting new parts. We have to be careful not to damage anything else that makes the computer work.
+The cheapest option is most likely to buy compatible new parts for our existing
+computer. However, we've never attempted tinkering with hardware in our computer
+before, and after buying all the tools and parts, we follow an instructional
+video to make the upgrades. In this case, we only have one computer to work on,
+and nowhere to test out opening up a computer and connecting new parts. We have
+to be careful not to damage anything else that makes the computer work.
 
-Lukcily, with code we have a "sandbox", or a safe place, to experiment with different outcomes. With `git` and GitHub, we're given "branching" as an option so that we can _branch out_ from our original repo and any working code it contains, and make changes separately. If the code works out, we can keep it! If not, we can throw it out and start fresh. What a relief!
+It would be nice if we could test out our computer upgrading skills first.
+Luckily, we _do_ have this option when coding!
 
 ## Explain the Primary Role of Branching
 
-Play off of this reference:
-"Wouldn't it be great if you could try things out in "sandbox" and then throw it away, with no consequence, if it didn't work out? "Branches" in git allow exactly this. You can propose "parallel universes" of code that can be compared, discussed, tried out, and ultimately imported or thrown away."
+If we had an identical computer that we could experiment, we could worry less
+about making any mistakes that could impact our working machine. This is the
+exact concept of branching.
```

Comment:
This paragraph might go better above the heading, then start this section with the next paragraph.